### DISCUSSION TITLE
Can you give clear my doubt about the noise stuff?

### DISCUSSION THREAD

**Discussion by gilroff** [ljleb/sd-webui-neutral-prompt#24] [Q&A] [answered]
I find it kinda hard to grasp the noise stuff. Isn't noise random? So AND PERP adds noise from its prompt, isnt it adding random stuff?
Or does noise mean AND PERP prompt's information in latent space in this context?
If I want to prompt for a cartoon character and later add photographic skin and lighting, Any recommendations on how to do that?
Thanks for the awesome extension<3

**Comment by ljleb** [comment, marked as answer]
> Isn't noise random?

In the readme, by "noise" I meant to refer to the latent noise returned by the SD model. A diffusion model returns the latent noise from a latent image, which is then subtracted from the previous iteration result. Maybe I should update the readme to refer less to technical terms or concepts where possible.

> If I want to prompt for a cartoon character and later add photographic skin and lighting, Any recommendations on how to do that?

I'd like to suggest latent interpolation for that, however this is not yet possible with this extension. In the furture I want to combine this extension and prompt fusion into a single extension to make it possible to interpolate the latent noises the model generates at each generation step.

For now, you can use prompt editing or prompt fusion to achieve these results. It's not ideal but you should get acceptable results with prompt editing at least.

**Reply by gilroff** [reply]
Yes, I think I understand things now. thanks for the help.
Yes, I think you should make a title like "Layman explanation" and put the practical idea using familiar concepts wherever needed.
Also I checked out prompt fusion. if both extensions are combined it will be a very powerful tool!
I have another doubt:
What method should I use for these:
1. Maximum number of elements derived from my general prompt and show up in the output over time.
2. Maximum variety in the output like poses and clothes without changing the prompt.
